backend/apps/vacancies/models.py

```diff
@@ -0,0 +1,131 @@
+from apps.accounts.models import User
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from taggit.managers import TaggableManager
+
+
+class EmploymentChoices(models.TextChoices):
+
+    FULL_TIME = "FT", _("Full time")
+    PART_TIME = "PT", _("Part time")
+    OTHER = "OT", _("Other")
```

Comment:
How about adding 'INTERNERSHIP' and 'VOLUNTEER'?

Reply:
I also put these choice classes into a seperate file in the next pull request, so you need to change imports after or wait for my pull request to be merged first and then rebase your branch so you don't forget.

Reply:
> I also put these choice classes into a seperate file in the next pull request, so you need to change imports after or wait for my pull request to be merged first and then rebase your branch so you don't forget.

I thought, maybe later I will fix vacancies and other models with a hotfix

Reply:
> How about adding 'INTERNERSHIP' and 'VOLUNTEER'?

This is a slightly different grouping. EmploymentChoices is how often it will be necessary to work on this vacancy. INTERNERSHIP can be like fulltime or parttime. This is another category, closer to a grade.

VOLUNTEER also can be like fulltime or parttime.